packages/devtools/src/launcher.js

```diff
@@ -22,6 +21,7 @@ import {
 
 const log = logger('devtools')
 
+const devicesMap = puppeteer.devices
```

Comment:
It seems that we are only accessing the device values, so I would suggest to get the values here already:
```suggestion
const devicesMap = Object.values(devicesMap)
```

Reply:
Done